docs/datagen/server/glm.md

```diff
@@ -0,0 +1,20 @@
+Global Loot Modifier Generation
+===============================
+
+Global Loot Modifiers (GLMs) can be generated for a mod by subclassing `GlobalLootModifierProvider` and implementing `#start`. Each GLM can be added generated by calling `#add` and specifying the name of the modifier, the [serializer of the modifier][serializer], and the [modifier instance][instance] to be serialized. After implementation, the provider must be [added][datagen] to the `DataGenerator`.
```

Comment:
Should there be a link to the GLM page at `Global Loot Modifiers (GLMs)`?

Reply:
And now it is.